src/main/java/duke/TaskManager.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.tasks.Task;
+
+public class TaskManager {
+    private ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public TaskManager() { }
+    public TaskManager(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTask(Task t) {
```

Comment:
These methods are missing their javadoc header comments. If you have time, I'd recommend adding them because it really makes the code easier to read.